components/class-go-newrelic-apm.php

```diff
@@ -81,7 +82,21 @@ public function init()
 			newrelic_set_user_attributes( 'not-logged-in', '', 'no-role' );
 		}// END else
 	}// END init
+	public function metabox()
+	{
+
+		$checked = ! get_post_meta( $post_id, 'go_newrelic_disable', TRUE )  ? TRUE : FALSE;
```

Comment:
can we make this truthy?
`$checked = get_post_meta( $post_id, 'go_newrelic_disable', TRUE )  ? FALSE: TRUE;`